language_sentiment_text.rb

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "language_sentiment_text")
+
+# sample-metadata
+#   title: Analyzing Sentiment
+#   description: Analyzing Sentiment in a String
+#   bundle exec ruby samples/v1/language_sentiment_text.rb [--text_content "I am so happy and joyful."]
+
+require "google/cloud/language"
+
+# [START language_sentiment_text]
+
+# Analyzing Sentiment in a String
+#
+# @param text_content {String} The text content to analyze
+def sample_analyze_sentiment text_content
+  # Instantiate a client
+  language_client = Google::Cloud::Language.new version: :v1
+
+  # text_content = "I am so happy and joyful."
+
+  # Available types: PLAIN_TEXT, HTML
+  type = :PLAIN_TEXT
+
+  # Optional. If not specified, the language is automatically detected.
+  # For list of supported languages:
+  # https://cloud.google.com/natural-language/docs/languages
+  language = "en"
+  document = {
```

Comment:
Newline between the setting of the constituent fields and the setting of the request hash. Otherwise it looks like the last comment (describing `language`) also applies to `document`.

Reply:
Hmm.

If the issue is with a comment appearing to talk about N variable below the comment (rather than just the 1 immediately below), then is the requirement:
 - for every field/variable assignment with a comment, add a newline after the field/variable assignment

^--- can do this!